examples/angular/README.md

```diff
@@ -0,0 +1,44 @@
+# Mediainfo Angular example
+
+**Step #1**
+
+Copy the node_modules/mediainfo.js/dist/MediaInfoModule.wasm to your project's src directory,
+
```

Comment:
I forgot to remove the no more necessary step 1. If you decide to accept the PR could you please remove it ? Thanks.